concent_api/core/message_handlers.py

```diff
@@ -671,6 +676,11 @@ def handle_send_force_payment(client_message: message.concents.ForcePayment) ->
             reason = message.concents.ServiceRefused.REASON.DuplicateRequest
         )
 
+    if not are_eth_addresses_and_singularities_equal_in_all_subtask_results_accepted_list(client_message):
+        return message.concents.ServiceRefused(
+            reason = message.concents.ServiceRefused.REASON.DuplicateRequest
```

Comment:
REASON will be changed. I wait until Golem add new REASON and I will change it. But you can give me your feedback for the rest of code now.

Reply:
changed according to @lukasz-glen suggestion